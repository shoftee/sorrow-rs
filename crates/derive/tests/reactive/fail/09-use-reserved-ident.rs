use sorrow_derive::Reactive;

#[derive(Reactive)]
struct OuterTest {
    __runtime: i32,
}

fn main() {}
