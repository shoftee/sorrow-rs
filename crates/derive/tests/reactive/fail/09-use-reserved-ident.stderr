error: derive(Reactive) uses this identifier internally, please use another one.
 --> tests/reactive/fail/09-use-reserved-ident.rs:5:5
  |
5 |     __runtime: i32,
  |     ^^^^^^^^^
